packages/catalog-model/src/entity/policies/FieldFormatEntityPolicy.test.ts

```diff
@@ -65,11 +65,6 @@ describe('FieldFormatEntityPolicy', () => {
     await expect(policy.enforce(data)).rejects.toThrow(/kind/);
   });
 
-  it('handles missing metadata gracefully', async () => {
-    delete data.medatata;
-    await expect(policy.enforce(data)).resolves.toBe(data);
-  });
```

Comment:
entities require metadata, so they shouldn't handle it being missing gracefully.  test only passed because metadata was misspelled?

Reply:
alternatively, could change this to throw an error if it's missing, just rewriting the test to reject missing metadata.

Reply:
I think it's fine to drop 👍

It just throws "cannot read ... of undefined" anyway, so it's not really handling it at all. Metadata is required in the TS type too and it's up to the catalog processing engine to ensure that it exists.